### PR TITLE
Logging

### PR DESCRIPTION
**Pull request by aktrinh17**
Implemented basic json parsing library for logging.

### CODE REVIEW

File: leveldb/Log/json/jsonParsingExample.cpp [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+  cout << "Printing File Contents:\n";
+  printFileContents(logFile);
+
+  cout << "printing value from key 0 " << getValueFromKey(logFile, "Timestamp0") << endl;
+  cout << "printing value from key 1 " << getValueFromKey(logFile, "Timestamp1") << endl;
+  cout << "printing value from key 2 " << getValueFromKey(logFile, "Timestamp2") << endl;
+  cout << "printing value from key 3 " << getValueFromKey(logFile, "Timestamp3") << endl;
+  cout << "printing value from key 4 " << getValueFromKey(logFile, "Timestamp4") << endl;
+
```
Comment (zhou-1) [inline leveldb/Log/json/jsonParsingExample.cpp:31]: Good examples!

**Review by zhou-1 — COMMENTED** [review]
Good job!

**Review by wfystx — APPROVED** [review]
well done

**Review by thezhe — APPROVED** [review]
Looks good.

**Pull request merged**